### ISSUE TITLE
Verbose of model.fit can not display the loss of Custom loss function

### ISSUE THREAD

**Issue by triper1022** [keras-team/keras#15230]
**System information**.
- Have I written custom code (as opposed to using a stock example script provided in Keras):
YES, I ran it on [colab tpu runtime(shared notebook)](https://colab.research.google.com/drive/1Et1psrCxIBf9q_tPIN0ki3_i-YSW2eH_?usp=sharing).

- TensorFlow version (use command below):
2.6
- Python version:
3.7

**Describe the problem**.
**Describe the current behavior**.
I created a custom loss function in keras, 
and I have tested it in all model settings and loss function,
with one sample in my dataset and it can raise correct loss value.
(it shows in notebook)

But when I compiled the model and passed into tpu strategy,
the verbose displays nan.
I have tried to wrap my loss function with
tf.keras.losses.Loss or tf.keras.losses.LossFunctionWrapper.
Neither of them worked.

**Describe the expected behavior**.

Please help me make it work

**Standalone code to reproduce the issue**.
[a link to Colab/Jupyter/any notebook](https://colab.research.google.com/drive/1Et1psrCxIBf9q_tPIN0ki3_i-YSW2eH_?usp=sharing).

**Comment by harishsg99** [comment]
You can use tf.keras.compile(loss= custom loss function name , optimizer = optimizer)
